TypeCobol.Test/Parser/Programs/TypeCobol/FunctionsDeclaration.rdz.tcbl

```diff
@@ -111,9 +111,6 @@
        
        DECLARE FUNCTION FunConditions PRIVATE
              INPUT  gender PIC X(01)
-                 88  valid-gender VALUE 'F' 'M'
-                 88  female VALUE 'F'
-                 88  male   VALUE 'M'
```

Comment:
We should keep the test here too

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1669/commits/3c0f9d082a18ff1b641cb676ce9152a389f4f984